.gitignore

```
@@ -2,4 +2,6 @@
 .DS_Store
 .vscode
 target
-.metahash
\ No newline at end of file
+.metahash
+*.meta.txt
+meta.txt
```

```suggestion
meta.txt
```
`*.meta.txt` is enough.